stackable-base/Dockerfile

```diff
@@ -5,6 +5,14 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9@sha256:87bcbfedfd70e67aab38
 # intentionally unused
 ARG PRODUCT
 
+# Sets the default shell to Bash with strict error handling and robust pipeline processing.
+# "-e": Exits immediately if a command exits with a non-zero status
+# "-u": Treats unset variables as an error, preventing unexpected behavior from undefined variables.
+# "-o pipefail": Causes a pipeline to return the exit status of the last command in the pipe that failed, ensuring errors in any part of a pipeline are not ignored.
+# "-c": Allows the execution of commands passed as a string
+# This is automatically inherited in all other Dockerfiles that use this unless it is overwritten
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
```

Comment:
**praise**: I also like this explanation of the flags/args.

Reply:
Thanks. As we only have this three times or so now I decided to include it. And it also contains the sentence about the inheritance.